xla/service/gpu/runtime3/copy_thunk.cc

```diff
@@ -22,29 +22,50 @@ limitations under the License.
 #include "xla/service/gpu/thunk.h"
 #include "xla/status.h"
 #include "xla/stream_executor/stream_executor.h"
+#include "xla/service/gpu/ir_emission_utils.h"
 
 namespace xla {
 namespace gpu {
 
 DeviceToDeviceCopyThunk::DeviceToDeviceCopyThunk(
```

Comment:
Seems like "DeviceToDeviceCopy" is no longer an appropriate name

Reply:
OK, I will create DeviceToHostCopy/HostToDeviceCopy thunks, leaving DeviceToDeviceCopyThunk alone, as it is used in other places.